packages/metascraper-helpers/test/index.js

```diff
@@ -210,7 +210,7 @@ describe('metascraper-helpers', () => {
       )
     ).be.equal(
-      'Let me tell you the story of two investors, neither of whom knew each other, but whose paths crossed in an interesting way. Grace Groner was orphaned at age 12. She never married. She never had kids. She never drove a car. She lived most of her life alone in a one-bedroom house and worked her whole…'
+      'Let me tell you the story of two investors, neither of whom knew each other, but whose paths crossed in an interesting way. Grace Groner was orphaned at age 12. She never married. She never had kids. She never drove a car. She lived most of her life alone in a one-bedroom house and worked her whole …'
```

Comment:
I think the "…" need to be closed to the latest word to be grammatically valid

Reply:
I just reverted this line to how it was before #449 : https://github.com/microlinkhq/metascraper/commit/5e386b98c6b4b26a837e524bd8ad3b20c7eafee6
Neither node-truncate nor this new truncation method account for grammatical validity; that space is only present because it's the 300th character in the string.

Reply:
why not `.trim()` then before appending `…`

Reply:
That would be a new behavior, not present in the original node-truncate: https://github.com/FGRibreau/node-truncate/blob/a92d6b66f5a71c33e52484862ed069fa12411b24/truncate.js#L51-L52

So if someone has a test like the one in`metascraper-helpers/test/index.js`, it would start failing since trailing spaces have historically been retained (before #449).

Since we're already breaking URLs apart, should we minimize further disruption?

Reply:
this is going to be shipped as a minor release so people can take into considerations if something breaks

Reply:
Ah okay. My day-job is a sysadmin, so I tend to fear breaking changes haha!

Pushed the change.